ui/lib/sync/addon/components/secrets/page/destinations.hbs

```diff
@@ -67,44 +67,44 @@
           </code>
         </Item.content>
 
-        <Item.menu>
-          {{#if destination.destinationPath.isLoading}}
-            <li class="action">
-              <LoadingDropdownOption />
-            </li>
-          {{else}}
-            <li>
-              <LinkTo
-                class="has-text-black has-text-weight-semibold"
+        <Item.menu @hasMenu={{false}}>
```

Comment:
`@hasMenu` is false so that we can use the `Hds::Dropdown` istead of `PopupMenu`